crates/modelardb_server/src/metadata/mod.rs

```diff
@@ -557,17 +516,26 @@ impl MetadataManager {
         let min_value = Self::rewrite_special_value_to_normal_value(compressed_file.min_value);
         let max_value = Self::rewrite_special_value_to_normal_value(compressed_file.max_value);
 
+        let created_at = compressed_file
+            .file_metadata
+            .last_modified
+            .timestamp_millis();
+
         // query_schema_index is simply cast as a model table contains at most 1024 columns.
+        // size is simply cast as it is unrealistic for a file to use more bytes than the max value
```

Comment:
Very very minor so I Approved the PR, but maybe change to "Size is simply..."

Reply:
Maybe it is best to keep it as `size` since it is the field `size` that is referred to just like with `query_schema_index` above.

Reply:
Makes sense, I misread the sentence and did not realize it was the name of a field.